src/m365/cli/commands/config/config-reset.ts

```diff
@@ -0,0 +1,67 @@
+import { Cli, Logger } from "../../../../cli";
+import { CommandOption } from "../../../../Command";
+import GlobalOptions from "../../../../GlobalOptions";
+import { settingsNames } from "../../../../settingsNames";
+import AnonymousCommand from "../../../base/AnonymousCommand";
+import commands from "../../commands";
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  key?: string;
+}
+
+class CliConfigResetCommand extends AnonymousCommand {
+  private static readonly optionNames: string[] = Object.getOwnPropertyNames(settingsNames);
+
+  public get name(): string {
+    return commands.CONFIG_RESET;
+  }
+
+  public get description(): string {
+    return 'Resets the specified CLI configuration option to its default value';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.key = (!(!args.options.key)).toString();
```

Comment:
It's perfectly fine for us to track the actual key as it's a list of fixed values